pkg/util/runtimeconfig/manager_test.go

```diff
@@ -35,10 +38,10 @@ func (l *TestLimits) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return unmarshal((*plain)(l))
 }
 
-func testLoadOverrides(filename string) (interface{}, error) {
+func testLoadOverrides(filename string) (interface{}, []byte, error) {
```

Comment:
That is quite a lot of code duplication with `loadRuntimeConfig`, maybe with a func argument we could get create a exported method

Reply:
Reason for that duplication is to 1) avoid inverse package dependency, 2) make it possible to use different structures to load config file into.

This code does *not* belong to `runtimeconfig.Manager`, but there can be a helper function to load YAML into specified structure and return its hash as well.